src/lib/holocene/tab/tab.svelte

```diff
@@ -64,13 +64,13 @@
 
 <style lang="postcss">
   .tab {
-    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm hover:border-b-2 hover:border-blue-700 md:text-base;
+    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm outline-none hover:text-brand focus-visible:rounded-sm focus-visible:shadow-focus focus-visible:shadow-indigo-300 dark:text-secondary hover:dark:text-brand;
 
     border-color: transparent;
   }
 
   .tab.active {
-    @apply border-b-2 border-blue-700 font-medium text-blue-700;
+    @apply border-b-2 border-information font-medium text-primary hover:text-brand focus-visible:shadow-focus focus-visible:shadow-indigo-300 dark:text-secondary hover:dark:text-brand;
```

Comment:
I'm not sure why we need the focus states defined here again? Also I think we can move the `font-medium` to the main .tab style definition so that the font size doesn't change whether the tab is active. Also I think the `hover:dark:text-brand` is redundant here as well.